### PR TITLE
Added structure for new evaluation

### PR DESCRIPTION
**Pull request by bojkomatias**
## Hello!

_This PR Also includes the #148 issue_ 
**Be sure to test it with develop database**

- Added a component <ReviewQuestion/> to be included in the form. This component is used to map through the questions a review holds (when created/assigned). Reads/Maps the correct question through a find(x=>x) method, and enables a radio selector (TRUE, FALSE) with optional comment if FALSE.
- Also saw that I needed to change Schema (planned in 148, but fixed here for easier testing). A structure for Question was added, and the **data** replaced for an array of **questions** in the review model.
- Added visualization in <ReviewQuestionView /> component, so overall structure stayed the same, just replaced the text-box, with a map of this component.

> All the questions are initially on the config folder. But the structure truly resembles one of a database, so later on, we could drop the questions there and it would just work.

### CODE REVIEW

**Review by amilcarrey — CHANGES_REQUESTED** [review]
Como siempre, una delicia esto. Hay detalles chiquitos pero estoy abierto a escuchar tu feedback de mi feedback 😅

File: src/modules/review/elements/review-form.tsx [hunk 1]
```diff
@@ -60,151 +57,266 @@ export default function ReviewForm({
...
+                                    ? 'hidden'
+                                    : ''
+                            }
+                            onClick={() =>
+                                addReview(
+                                    {
+                                        ...form.values,
+                                        verdict: 'NOT_REVIEWED',
+                                    },
+                                    true
```
[the remark this answers is not in the document]
Comment (bojkomatias) [reply src/modules/review/elements/review-form.tsx]: Dale, cambie por el texto

File: src/modules/review/elements/review-item.tsx [hunk 2]
```diff
@@ -92,17 +92,13 @@ export default function ReviewItem({
...
-                    //     hidden: review.revised,
-                    //     block: !review.revised,
-                    // })}
-                    >
-                        <TextItemView
-                            title=""
-                            content={review.data}
-                            className="px-2"
-                        />
+                    <div className="max-h-[70svh] overflow-y-auto">
```
[the remark this answers is not in the document]
Comment (bojkomatias) [reply src/modules/review/elements/review-item.tsx]: Collapse agregado, con mas padding y tambien agregue la numeracion de preguntas aca.

File: src/modules/review/elements/review-question.tsx [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+                        form.setFieldValue(
+                            'questions.' + index + '.comment',
+                            ''
+                        )
+                }}
+                defaultValue={
+                    form.getInputProps('questions.' + index + '.approved').value
+                }
+            >
+                <RadioGroup.Label className="select-none text-xs">
```
[the remark this answers is not in the document]
Comment (bojkomatias) [reply src/modules/review/elements/review-question.tsx:34]: Agregado!

**Review by amilcarrey — APPROVED** [review]

**Pull request merged**